src/qute/languageServer/client.ts

```diff
@@ -2,10 +2,12 @@ import * as requirements from './requirements';
 
 import { DidChangeConfigurationNotification, LanguageClientOptions } from 'vscode-languageclient';
 import { LanguageClient } from 'vscode-languageclient/node';
-import { ExtensionContext, commands, workspace } from 'vscode';
+import { ExtensionContext, commands, workspace, window, ConfigurationTarget } from 'vscode';
 import { prepareExecutable } from './javaServerStarter';
 import { registerVSCodeQuteCommands } from '../commands/registerCommands';
 import { QuteClientCommandConstants } from '../commands/commandConstants';
+import { QuarkusContext } from '../../QuarkusContext';
```

Comment:
remove this import. The main idea is to use the context coming from connectToQuteLS parameter.